joern-cli/frontends/jimple2cpg/src/test/scala/io/joern/jimple2cpg/querying/MemberTests.scala

```diff
@@ -24,4 +24,28 @@ class MemberTests extends JimpleCode2CpgFixture {
     val List(x) = cpg.member.typeDecl.l
     x.name shouldBe "Foo"
   }
+
+  val cpg2 = code("""
+      |import java.util.Objects;
+      |class ModifiersTest {
+      |    private static final String finalPrivateString = "PRIVATE_STATIC_FINAL";
+      |    public static final String finalPublicString = "PUBLIC_STATIC_FINAL";
+      |    public String publicString;
+      |    public static String staticString = "PUBLIC_STATIC";
+      |    private static final Object obj = new Object();
+      |    public static final int FLAG = 1;
+      |    public static int staticInt = 100;
+      |    protected int mInt;
+      |    long mLong;
+      |}
+      |""".stripMargin).cpg
+  "should have modifiers for MEMBER" in {
+    val members                 = cpg2.typeDecl.name("ModifiersTest").member.l
+    val finalPrivateStringField = members.find(_.name == "finalPrivateString").get
+    finalPrivateStringField.modifier.map(_.code).mkString(" ") shouldBe "private static final"
```

Comment:
This might be a safer check
```suggestion
    finalPrivateStringField.modifier.map(_.code) shouldBe List("private", "static", "final")
```